cirq/google/engine/engine.py

```diff
@@ -152,47 +156,43 @@ class Engine:
     """
 
     def __init__(self,
-                 api_key: Optional[str] = None,
-                 api: str = 'quantum',
+                 project_id: str,
                  version: str = 'v1alpha1',
-                 default_project_id: Optional[str] = None,
                  discovery_url: Optional[str] = None,
                  default_gcs_prefix: Optional[str] = None,
                  proto_version: ProtoVersion = ProtoVersion.V1,
                  **kwargs) -> None:
```

Comment:
Do we still need this kwargs argument? It strikes me as very "asking for what you want to make what you want" instead of "ask for what you need".

Reply:
We use this today for alternative auth configurations that haven't been documented and I'm not sure we'll explicitly try to manage with our interface. We could point to the googleapiclient docs if that's useful?

Reply:
I think we do need it but we should better document what it is.  Could you add a doc string.  Maybe also just call it service_args.  Also it could just be a dict and not a **kwargs, I guess.

Reply:
Makes sense, I added a dict service_args and documented it.